docs/resources/shadow.md.erb

```diff
@@ -81,7 +81,7 @@ Use `where` to match any of the supported [filter criteria](#filter_criteria). `
 
 ## Properties
 
-As a [plural resource](https://www.inspec.io/docs/reference/glossary/#plural_resource), all of `shadow`'s properties return lists (that is, Ruby arrays). `include` and `be_empty` are two useful matchers when working with lists. You can also perform manipulation of the lists, such as calling `uniq`, `sort`, `count`, `first`, `last`, `min`, and `max`.
+As a [plural resource](https://www.inspec.io/docs/reference/glossary/#plural_resource), all of `shadow`'s properties return lists (that is, Ruby Arrays). `include` and `be_empty` are two useful matchers when working with lists. You can also perform manipulation of the lists, such as calling `uniq`, `sort`, `count`, `first`, `last`, `min`, and `max`.
```

Comment:
Data types are not proper nouns. I won't die on this hill, though.

Reply:
Ruby classes are proper nouns. Here I'm referring to the Ruby [Array](http://ruby-doc.org/core-2.4.2/Array.html) class.

Reply:
http://web.mit.edu/course/21/21.guide/capitals.htm , see 
_General Guidelines for Capitalizing Scientific Terms_. But as I said, this really isn't my hill.